File: src/main/java/com/gitlabci/plugin/language/pasrser/psi/GitlabYamlTokenType.java
package com.gitlabci.plugin.language.pasrser.psi;

import com.gitlabci.plugin.language.GitlabYamlLanguage;
import com.intellij.psi.tree.IElementType;

import org.jetbrains.annotations.NonNls;
import org.jetbrains.annotations.NotNull;

public class GitlabYamlTokenType extends IElementType {
    public GitlabYamlTokenType(@NonNls @NotNull String debugName) {
        super(debugName, GitlabYamlLanguage.INSTANCE);
    }

    public String toString() {
        return "GitlabKeywordTokenType." + super.toString();
    }
}

File: src/main/kotlin/com/github/fiskirton/gitlabciplugin/language/psi/GitlabYamlTokenType.kt
package com.github.fiskirton.gitlabciplugin.language.psi

import com.github.fiskirton.gitlabciplugin.language.GitlabYamlLanguage
import com.intellij.psi.tree.IElementType

import org.jetbrains.annotations.NonNls

class GitlabYamlTokenType(debugName: @NonNls String) : IElementType(debugName, GitlabYamlLanguage.INSTANCE) {
    override fun toString(): String {
        return "GitlabKeywordTokenType." + super.toString()
    }
}